### DISCUSSION TITLE
[Feature request] Tabbed lists

### DISCUSSION THREAD

**Discussion by mgrimace** [sbondCo/Watcharr#553] [Ideas]
Hi,
Love watcharr, and my humble request would be for tabbed lists along the top. Specifically, myuse-case is separate lists for myself, to-watch-with-my-spouse, for-the-kids, etc. Terrible mockup below. 

I should mention that I mean this as different than a separate spouse or kids account. In this case, I want to track what I want to watch with my spouse, for example, which is very different than the shows she would watch on her own if that makes sense.

![Screenshot 2024-06-21 094727](https://github.com/sbondCo/Watcharr/assets/55518507/1e887f97-f065-49ea-9f3b-2b46ad2634ba)

Thank you for all your work on this!

**Comment by IRHM** [comment]
Hey @mgrimace, glad you're liking Watcharr!

This sounds like we could have these tabs as an add-on to [Give watched content tags #199](https://github.com/sbondCo/Watcharr/issues/199).

The issue above would mean you'd have a main watchlist that has everything, then you could tag individual shows/movies with `me` or `kids`, which we could show tabs for to filter.

How does that sound?

Thanks!

p.s. thanks for the mockup I now own 100% rights to that design and will be implementing an exact copy of it.

**Reply by mgrimace** [reply]
Thanks, that makes sense! If I'm understanding correctly it sounds very similar to how I use virtual libraries in Calibre (books).
A 'main' unfiltered view for all content, with support for multiple tags (`kids`, `wife`, etc.) as per #199 (which would be really cool), then, ideally, adding to #199 would be support for a tabbed view main page, where each new tab can be customized with one or more tags.  For example `me` + `spouse` tags for the tab `watch together`. 

Ha, and you hereby have full rights to my mockup! Just imagine I added another transparent tab with a `+` and clicking that would prompt the user to select tags, add a name for the tab, etc.

**Reply by IRHM** [reply]
Yup that sounds correct. I haven't used calibre before, but creating custom pages/views with multiple custom tags sounds very cool! We could extend this by having preset filters that save (like saving the filter `SHOWS` and naming the tab shows to watch together, though that is less important than the tags).
